### PR TITLE
Changed the image build and its dir structure

### PR DESCRIPTION
**Pull request by petr-tik**
Removed one extra layer of directories.
Before:
/home/ffig/ffig-master
/home/ffig/flask

Now:
/home/ffig-master/
/home/flask

pip2 install to guarantee installing flask for py2

remove RUN .. curl, because curl wasn't installed - docker build was breaking
Instead ADD from the github link

Won't expose anymore - we need port binding in
docker run -p 5000:5000, not exposing portts

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,20 +8,17 @@ MAINTAINER support@ffig.org
...
     pip3 install flask
 
 # Install the ffig codebase. Use `cd` here to avoid several WORKDIR layers.
-RUN cd /home/ffig && \
-    curl -O https://github.com/FFIG/ffig/archive/master.zip && \
-    unzip ffig-master.zip && \
-    rm -f ffig-master.zip
+ADD https://github.com/FFIG/ffig/archive/master.zip /home/
+RUN unzip home/master.zip -d home/ && \
+    rm -f home/master.zip
```
Comment (ajbennieston) [inline Dockerfile]: We should install `curl` into the base image. This change makes two layers in this image, but the base image already has an `apt-get install ...` that we can just add `curl` to.

**Review by ajbennieston — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (petr-tik) [reply Dockerfile]: Are you happy with the rest?

Comment (ajbennieston) [reply Dockerfile]: Yeah. If I didn't comment on it, I'm happy with it.

**Review by ajbennieston — DISMISSED** [review]

**Comment by ajbennieston** [comment]
Sorry; approved too soon. Need to change the ADD/RUN back to a single RUN once the base image has been re-built.

File: Dockerfile [hunk 2]
```diff
@@ -13,9 +13,10 @@ RUN pip install --upgrade pip && \
     pip3 install flask
 
 # Install the ffig codebase. Use `cd` here to avoid several WORKDIR layers.
-ADD https://github.com/FFIG/ffig/archive/master.zip /home/
-RUN unzip home/master.zip -d home/ && \
-    rm -f home/master.zip
+RUN cd /home/ffig && \
+    curl -O https://github.com/FFIG/ffig/archive/master.zip && \
+    unzip ffig-master.zip && \
```
Comment (ajbennieston) [inline Dockerfile]: just `master.zip`?

**Comment by petr-tik** [comment]
@ajbennieston @jbcoe please review the final changes

**Review by ajbennieston — APPROVED** [review]

**Pull request merged**